arm/Microsoft.Synapse/workspaces/.parameters/basic.parameters.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "sxx-az-synws-basic-001"
+    },
+    "defaultDataLakeStorageAccountName": {
+      "value": "adpsxxazsaweux004"
+    },
+    "defaultDataLakeStorageFilesystem": {
+      "value": "synapsews"
+    },
+    "sqlAdministratorLogin": {
+      "value": "synwsadmin"
+    },
+    "initialWorkspaceAdminObjectId": {
+      "value": "<<deploymentSpId>>"
+    },
+    "userAssignedIdentities": {
+      "value": {
+        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-sxx-az-msi-x-001": {}
+      }
+    },
+    "roleAssignments": {
+      "value": [
+        {
+          "roleDefinitionIdOrName": "Reader",
+          "principalIds": ["<<deploymentSpId>>"]
+        }
+      ]
+    },
+    "diagnosticLogsRetentionInDays": {
+      "value": 7
+    },
+    "diagnosticStorageAccountId": {
+      "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Storage/storageAccounts/adpsxxazsalaw001"
+    },
+    "workspaceId": {
+      "value": "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/microsoft.operationalinsights/workspaces/adp-sxx-az-law-x-001"
+    },
+    "eventHubAuthorizationRuleId": {
+      "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.EventHub/namespaces/adp-sxx-az-evhns-x-001/AuthorizationRules/RootManageSharedAccessKey"
+    },
+    "eventHubName": {
+      "value": "adp-sxx-az-evh-x-001"
+    },
+    "logsToEnable": {
+      "value": [
+        "SynapseRbacOperations",
+        "GatewayApiRequests",
+        "BuiltinSqlReqsEnded",
+        "IntegrationPipelineRuns",
+        "IntegrationActivityRuns",
+        "IntegrationTriggerRuns"
+      ]
+    }
+  }
```

Comment:
Might be good idea to add an endpoint test here too. [Ref](https://github.com/Azure/ResourceModules/blob/main/arm/Microsoft.Automation/automationAccounts/.parameters/parameters.json)